recipes/earthengine-api/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "earthengine-api" %}
+{% set version = "0.1.137" %}
+{% set sha256 = "6dd2e0e8de8214c5cf499c55f13154f6a5109ef5ea673260c80b22f9a2b51c96" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  entry_points:
+    - earthengine = ee.cli.eecli:main
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - python
+    - pip
+    - google-api-python-client
```

Comment:
It looks like `google-api-python-client` and `pyopenssl` might not be needed as `build` requirements. If the `setup.py` doesn't import them directly or as a side-effect or list them under `setup_requires`, you won't need them for the build step.

Reply:
Removed suggested packages from build.